rpcs3/Emu/Cell/SPUThread.cpp

```diff
@@ -310,11 +310,29 @@ std::string SPUThread::get_name() const
 
 std::string SPUThread::dump() const
 {
-	std::string&& ret = cpu_thread::dump();
-	ret += fmt::format("\n" "Tag mask: 0x%08x\n" "MFC entries: %u\n", +ch_tag_mask, mfc_queue.size());
-	ret += "Registers:\n=========\n";
+	std::string ret = cpu_thread::dump();
 
-	for (uint i = 0; i<128; ++i) ret += fmt::format("GPR[%d] = %s\n", i, gpr[i]);
+	fmt::append(ret, "\nTag mask: 0x%08x", +ch_tag_mask);
+	fmt::append(ret, "\nMFC size: %u", mfc_size);
+
+	for (u32 i = 0; i < 16; i++)
+	{
+		if (i < mfc_size)
+		{
+			fmt::append(ret, "\n[%s t%02u]", mfc_queue[i].cmd, mfc_queue[i].tag);
```

Comment:
please drop this, we dont need to view the mfc commands on the debugger, theres visual studio debugger for this kinda stuff lol